mysql-test/suite/galera/t/galera_sst_failure.test

```diff
@@ -40,6 +40,55 @@ SET GLOBAL debug = "+d,wsrep_sst_donate_cb_fails";
 # cleanup
 SET GLOBAL debug = "-d,wsrep_sst_donate_cb_fails";
 
+# Setup node1 to skip sending backup on next SST
+SET GLOBAL debug = "+d,wsrep_sst_donor_skip";
+
+# Restart node 2 forcing SST
+--connection node_2
+
+# Remove the grastate.dat file to force an SST
+--remove_file $MYSQLTEST_VARDIR/mysqld.2/data/grastate.dat
+
+# Adjust SST timeout
+--exec cp $MYSQLTEST_VARDIR/my.cnf $MYSQLTEST_VARDIR/my_sst.cnf
+append_file $MYSQLTEST_VARDIR/my_sst.cnf;
+[sst]
+sst-timeout=20
```

Comment:
Proposal: name it sst-idle-timeout or sst-inactivity-timeout. Now it suggests that it is the timeout for the whole SST to finish.